### PR TITLE
fix: updating README.md and change slack message formatting

### PR DESCRIPTION
**Pull request by debabrata-shome**

### CODE REVIEW

**Review by LokeshNSF — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is a Heroku Buildpack you can add to your Heroku deployments that will notify a Slack channel every time your application is deployed.
 
-![screenshot](https://raw.githubusercontent.com/evantahler/heroku-buildpack-notify-slack-deploy/master/sceenshot.png)
+![screenshot](./slackimage.png)
```
Comment (KishoreNemalipuri) [inline README.md:5]: Like it! Thanks.

File: bin/compile [hunk 2]
```diff
@@ -17,8 +17,10 @@ export_env_dir() {
 export_env_dir $3
 
 SLACK_MESSAGE=$(cat <<-END
-Test: *$HEROKU_APP_NAME* was deployed.
-*Release notes links*:<https://github.com/salesforcedevs/developer-website/releases/latest|DSC>,<https://github.com/salesforcedevs/doc-framework-monorepo/releases/latest|SFDocs>
+*$HEROKU_APP_NAME* was deployed.
+*Release notes links*:
+•  <https://github.com/salesforcedevs/developer-website/releases/latest|DSC>
+•  <https://github.com/salesforcedevs/doc-framework-monorepo/releases/latest|SFDocs>
```
Comment (KishoreNemalipuri) [inline bin/compile:23]: Thanks  for the changes.

**Review by KishoreNemalipuri — COMMENTED** [review]
Awesome.

**Pull request merged**